include/deal.II/fe/fe_tools.h

```diff
@@ -777,6 +782,29 @@ namespace FETools
                     const DoFHandler<dim,spacedim> &dof2,
                     const ConstraintMatrix         &constraints,
                     OutVector                      &z2);
+
+  /**
+   * Same as above for external parallel VectorTypes
+   * on a parallel::distributed::Triangulation
+   */
+  template <int dim, class VectorType, int spacedim>
+  void extrapolate_parallel(const DoFHandler<dim,spacedim> &dof1,
```

Comment:
Or do a dispatch internally based on whether the DoFhandler is using a parallel triangulation or not?